src/Tokenizer/Tokens.php

```diff
@@ -1139,6 +1157,26 @@ public function clearTokenAndMergeSurroundingWhitespace($index)
     }
 
     /**
+     * Finds the line number of the first token previous to the one at the specified index, including itself.
+     *
+     * @param int $index Token index
+     *
+     * @return int Line number
+     */
+    private function findNearestPrevTokenLineNumber($index)
```

Comment:
Do you not understand or do you choose not to understand? As I [tried to explain](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2550#issuecomment-280913912), there are **two versions** of a file: the **original version** and the **fixed version**. We *do not* want to report the lines of the fixed version because they are ephemeral and meaningless to a user wishing to manually fix the original source file; they are only acquainted with the **original file**. Ergo it is not only a huge waste of time, effort and processing power to track the lines of the modified tokens but it is also **strictly incorrect** to do so.